src/Item.js

```diff
@@ -83,6 +117,29 @@ function Item({ userToken, item }) {
         />
         {itemName}
       </label>
+      {/* // [delete-item] 1. Create a button next to each item in list */}
+      <button type="button" onClick={handleOpenModal}>
+        X
+      </button>
+      {/* // [delete-item] 2. Create a button modal that will render when user tries to delete item */}
+      <Modal
+        open={openModal}
+        onClose={handleCloseModal}
+        aria-labelledby="simple-modal-title"
+      >
+        <StyledModalLayout>
+          <h1>
```

Comment:
```suggestion
          <h1 id="simple-modal-title">
```
Probably better would be `delete-modal-title` (same with the `aria-labelledby` above)